heron/common/src/cpp/metrics/metrics-mgr-st.cpp

```diff
@@ -55,6 +52,11 @@ MetricsMgrSt::~MetricsMgrSt() {
   }
 }
 
+void MetricsMgrSt::Start(const sp_string& _my_hostname, sp_int32 _my_port,
+     const sp_string& _component, const sp_string& _task_id,  EventLoop* eventLoop) {
+  client_ = new MetricsMgrClient(_my_hostname, _my_port, _component, _task_id, eventLoop, options);
```

Comment:
also eventloop should not necessary here. Its already passed to us in the constructor